packages/venia-concept/templates/open-document.mst

```diff
@@ -24,5 +24,5 @@
     {{> templates/default-font-include}}
     <link rel="preconnect" href="{{env.MAGENTO_BACKEND_URL}}">
     {{#bundles.prefetch}}
-    <link rel="prefetch script" href="{{.}}">
+    <link rel="prefetch script" href="/{{.}}">
```

Comment:
These values are provided by the `asset-manifest` which is created by the webpack asset manifest plugin. This plugin provides a `publicPath` prop which we should be using to prepend the forward slash rather than hardcoding in the templates.